dropwizard/src/test/java/com/linecorp/armeria/dropwizard/TestApplication.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.dropwizard;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.server.ServerBuilder;
+
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+
+public class TestApplication extends Application<TestConfiguration> {
+    @Nullable
+    ServerBuilder serverBuilder;
+
+    @Override
+    public void initialize(Bootstrap<TestConfiguration> bootstrap) {
+        final ArmeriaBundle<TestConfiguration> bundle =
+                new ArmeriaBundle<TestConfiguration>() {
+                    @Override
+                    public void configure(ServerBuilder builder) {
+                        builder.service("/armeria", (ctx, res) -> HttpResponse.of("Hello, Armeria!"));
+                        serverBuilder = builder;
```

Comment:
I'm not sure there's a good reason to persist this as a field. The builder is built after this method within the bundle. If anything, you could move the field within the anonymous class, but even there, I'm not sure what good it would do

Reply:
Thanks for pointing this. I added this for testing. It is not used anymore.